pages/instructorQuestionOverview/instructorQuestionOverview.ejs

```diff
@@ -0,0 +1,173 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <%- include('../partials/head'); %>
+    <script src="/javascripts/d3.min.js"></script>
+    <script src="/javascripts/lodash.min.js"></script>
+    <script src="/localscripts/stacked_histogram.js"></script>
+    <script src="/localscripts/histmini.js"></script>
+    <script>
+      document.urlPrefix = '<%= urlPrefix %>';
+    </script>
+    <style>
+      .popover{
+        max-width: 50%;
+      }
+    </style>
+  </head>
+  <body>
+    <script>
+      $(function() {
+        $('[data-toggle="popover"]').popover({
+          sanitize: false
+        });
+        // make the file inputs display the file name
+        $(document).on('change', '.custom-file-input', function () {
+            let fileName = $(this).val().replace(/\\/g, '/').replace(/.*\//, '');
+            $(this).parent('.custom-file').find('.custom-file-label').text(fileName);
+        });
+      });
+    </script>
+    <%- include('../partials/navbar'); %>
+    <div id="content" class="container-fluid">
+      <%- include('../partials/navtabs'); %>
+      <div class="card mb-4">
+        <div class="card-header bg-primary text-white d-flex">
+          Question <%= question.qid %>
+        </div>
+
+        <table class="table table-sm two-column-description">
+          <tbody>
+            <tr>
+              <th>Title</th>
+              <td><%= question.title %></td>
+            </tr>
+            <tr>
+              <th>QID</th>
+              <td>
+                  <span class="pr-2"><%= question.qid %></span>
+                  <% if ((authz_data.has_course_permission_edit) && (! course.options.isExampleCourse)) { %>
+                  <button id="changeQidButton" tabindex="0" class="btn btn-xs btn-secondary" type="button" data-toggle="popover" data-container="body" data-html="true" data-placement="auto" title="Change QID" data-content="<%= include('../partials/changeIdForm', {id_label: 'QID', buttonID: 'changeQidButton', id_old: question.qid, ids: qids}) %>" data-trigger="manual" onclick="$(this).popover('show')">
+                      <i class="fa fa-i-cursor"></i>
+                      <span>Change QID</span>
+                  </button>
+                  <% } %>
+                  <% if (questionGHLink) { %>(<a class="pl-2" target="_blank" href="<%= questionGHLink %>">view on GitHub</a>)<% } %>
```

Comment:
Nit: the `pl-2` has an unfortunate effect on spacing when the adjacent button isn't rendered:

<img width="495" alt="Screen Shot 2019-12-15 at 9 59 37 PM" src="https://user-images.githubusercontent.com/1476544/70882968-5166ed00-1f86-11ea-8198-8fdf5d74c2da.png">


I might suggest adding `mr-2` to the button instead.

Reply:
Good catch. Done.